components/camel-soap/src/main/java/org/apache/camel/dataformat/soap/Soap11DataFormatAdapter.java

```diff
@@ -199,8 +199,18 @@ private Exception createExceptionFromFault(Fault fault) {
                 throw new RuntimeCamelException(e);
             }
         }
+
+	Object detailObj = faultDetail.getAny().get(0);
+
+        if (!JAXBIntrospector.isElement(detailObj)) {
```

Comment:
This does not compile, `JAXBIntrospector.isElement` cannot be referenced as a static method, see [javadoc](https://docs.oracle.com/javaee/7/api/javax/xml/bind/JAXBIntrospector.html#isElement-java.lang.Object-).